deploy/helm/sumologic/templates/traces/otelcol/service.yaml

```diff
@@ -8,6 +8,11 @@ metadata:
     {{- include "sumologic.labels.scrape.traces" . | nindent 4 }}
     {{- include "sumologic.labels.common" . | nindent 4 }}
 spec:
+  # Otel DNS load balancing requires collector service to be headless
+  # in order to skip k8s load balancing and instead populate DNS records.
+  {{- if eq .Values.sumologic.traces.loadBalancing true }}
+  clusterIP: None
+  {{- end }}
```

Comment:
Could we add headless service as another template like for fluentd or otelcol (logs/metrics)?
https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/ec217e0513b647b150b9b5cd5baa6012c1fe88ea/deploy/helm/sumologic/templates/logs/fluentd

Reply:
Done.